optaplanner-website-root/content/blog/2023/04/05/BoostVRPListSelectorScalability.adoc

```diff
@@ -0,0 +1,76 @@
+= Boosting list variable scalability with nearby selection
+dupliaka
+2023-04-05
+:page-interpolate: true
+:jbake-type: post
+:jbake-tags: insight, design, vehicle routing. move selector
+:jbake-social_media_share_image: nearby-selection.png
+
+As a solver of optimization problems, the VRP ListSelector faces a significant challenge when it comes to scalability.
+With each additional node in the problem, the complexity of the algorithm increases exponentially, making it difficult to find feasible solutions within a reasonable amount of time.
+
+== How to address this challenge?
+
+Nearby selection has recently been implemented for the `PlanningListVariable` as a tool to reduce the amount of nodes that need to be considered during the optimization process. By pre-calculating a distance matrix and using different distributions to limit the number of moves that need to be considered, this selector improves the scalability of the algorithm, making it possible to tackle larger and more complex problems.
+
+But what is the underlying analysis behind the success of the `NearbyMoveSelector`, and where might bottlenecks arise in the optimization process?
+
+At its core, the `NearbyMoveSelector` is a tool for reducing the complexity of the optimization process. By limiting the number of moves that need to be considered, it reduces the number of comparisons required, ultimately leading to a reduction in execution time. This is achieved by pre-calculating a distance matrix, which allows the algorithm to quickly identify the closest nodes and *focus its attention on those that are most relevant*.
+Imagine you're a delivery company trying to deliver packages to different cities with the minimum driving time spent.
+Suppose you need to deliver a package from Berlin to Paris. Does it make sense to put a stop in Tokyo in the middle?
+
+== How does nearby selection improve the calculation?
+
+Let's compare the solver progress over time with `NearbyMoveSelector` and the default configuration represented by `Late
+Acceptance` with `</listChangeMoveSelector>` and `</listSwapMoveSelector>`.
+On the graphs below you can see the
+https://github.com/kiegroup/optaplanner/blob/main/optaplanner-examples/src/main/resources/org/optaplanner/examples/vehiclerouting/optional/benchmark/vehicleRoutingBenchmarkConfigListNearby.xml[optaplanner-benchmark]
+ results for different amount of samples. You can find out more about optaplanner-benchmarks https://www.optaplanner.org/docs/optaplanner/latest/benchmarking-and-tweaking/benchmarking-and-tweaking.html[here].
+
+== Execution time
+
+To analyze how much nearby reduces the
+time for finding the best solution lets compare the computational speed of the Late Acceptance (LA)
+algorithm with the Nearby algorithm, and analyze how quickly each algorithm
+improves the solution score. The results indicate that the Nearby algorithm
```

Comment:
I don't think it's a name of a well-known algorithm; thus I would not go for the capitals. More importantly, let's keep it consistent across the article.

```suggestion
improves the solution score. The results indicate that the nearby selection algorithm
```

Reply:
Added abbreviations
[d223fd8](https://github.com/kiegroup/optaplanner-website/pull/581/commits/d223fd82ab6cfb9c23e1a11ebec3fe75ea7fb69b)